lesson11/script.js

```diff
@@ -0,0 +1,50 @@
+function showLoadingImg(){
+    const loadingImg = document.createElement('img');
+    loadingImg.src = "loading-circle.gif";
+    loadingImg.alt = "ローディング画像";
+    document.getElementById('js-loading').appendChild(loadingImg);
+}
+
+function removeLoadingImg(){
+    document.getElementById('js-loading').remove();
+}
+
+async function getData(){
+    showLoadingImg();
+    try{
+        const url = "https://api.json-generator.com/templates/szdgGQcOLXuk/data?access_token=hu4bc7qh9znx2m8f53mn4mz2hryvdntkavwbw8j0";
+        const response = await fetch(url);
+        const json = await response.json();
+        const data = await json.data;
+        return data;
+    }catch(e){
+        document.getElementById('js-list').textContent = "データが取得できませんでした";
+        throw new Error(e);
+    }finally{
+        removeLoadingImg();
+    }
+}
+
+async function  showList(){
```

Comment:
IMO
[Moriken-sensei has talked](https://discord.com/channels/726252977928667179/807252588658688080/940254870194229288) about this before on Discord, and render seems to be the best practice.

```suggestion
async function  renderList(){
```

Reply:
Render is the perfect name for the function that creates the DOM.
I will rename!

Reply:
I renamed the function name.
https://github.com/nakagawamai/js-lesson/pull/13#discussion_r825410973